bsdploy/bootstrap-files/pkg.conf

```diff
@@ -1 +1,8 @@
 ASSUME_ALWAYS_YES: YES
+REPO_AUTOUPDATE: NO
```

Comment:
this should have been fixed with https://github.com/ployground/bsdploy/commit/35dbd924f4fa4432084c751cf0b346290eaa7bb2 can you double check?